.github/workflows/nm-build-test.yml

```diff
@@ -169,5 +169,16 @@ jobs:
             label: gcp-k8s-util
             timeout: ${{ inputs.build_timeout }}
             gitref: ${{ github.ref }}
-            python: ${{ inputs.python }}
+        secrets: inherit
+
+    # update docker
+    DOCKER:
+        needs: [BUILD, UPLOAD]
```

Comment:
we can just put this as "UPLOAD".

Reply:
We need BUILD here as we need the output whl value from the BUILD job.